apps/courses/src/components/forms/users/SignIn.tsx

```diff
@@ -61,11 +61,10 @@ export default ({ callback, ...props }: SignInFormProps) => {
   const onGithubSubmit = async () => {
     analytics.logEvent('Login', { method: 'github' });
 
-    const authUser = await auth
+    await auth
```

Comment:
`onGithubSubmit` uses async/await but `onSubmit` above does not. What do you think about using the same for these functions?